db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/GenericManagedCacheIFace.java

```diff
@@ -21,6 +25,8 @@
 
 public interface GenericManagedCacheIFace <K, V> {
     void put(K var1, V var2);
+    void putAll(ManagedCache<? extends K, ? extends V> var1);
     V getIfPresent(K k);
     void invalidate(K k);
+    Cache<K, V> getManagedCache();
```

Comment:
I added them in an earlier version of the code where I needed to copy an entire ManagedCache.  I just left the code in in case we have a need for it in the future.